roles/setup_efm/vars/EPAS.yml

```diff
@@ -26,3 +26,8 @@ pg_port: 5444
 
 # postgres service
 pg_service: "edb-as-{{ pg_version }}"
+
+# pgpool2 binary dir
+pgpool2_bin_path: "/usr/edb/pgpool{{ pgpool2_version }}/bin"
+pgpool2_version: 4.1
```

Comment:
pgpool2_version is not in PG.yml. Let's move this variable to defaults/main.yml

Reply:
Done